### ISSUE TITLE
Osm to network dataset but not vice versa. why?

### ISSUE THREAD

**Issue by coconutrees** [Esri/arcgis-osm-editor#180]
Hi Geeks!
I am able to convert osm to arcgis network dataset, however, I am not able to convert back from network dataset to osm?. Am I missing something?

**Comment by ThomasEmge** [comment]
No, you are not missing anything. The short answer would be:  because we haven't written the tool yet, and because no one has asked for such a thing. 

For the time being I marked it as an enhancement request.
